orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -113,7 +124,15 @@ class TaskController {
   @RequestMapping(value = "/pipelines/{id}/cancel", method = RequestMethod.PUT)
   @ResponseStatus(HttpStatus.ACCEPTED)
   void cancel(@PathVariable String id) {
-    executionRepository.cancel(id, AuthenticatedRequest.getSpinnakerUser().orElse("anonymous"))
+    def execution = executionRepository.retrievePipeline(id)
+    def user = AuthenticatedRequest.getSpinnakerUser().orElse("anonymous")
+    if (instanceStatusProvider.isInstanceUp(applicationName, execution.executingInstance)) {
+      executionRepository.cancel(id, user)
+    } else {
+      log.warn("Orca instance $execution.executingInstance running pipeline $id of $execution.application : $execution.name is no longer up... forcing cancelation")
+      executionRepository.forceCancel(id, user)
+      startTracker.triggerQueuedPipelines()
+    }
```

Comment:
If the Orca node that was running the job is not in service we should force cancellation & trigger the queue check. Otherwise proceed as before.